src/Cake.Svn/SvnTool.cs

```diff
@@ -33,5 +33,18 @@ protected ISvnClient GetClient()
         {
             return _clientFactoryMethod();
         }
+
+
+        /// <summary>
+        /// Checks the setting parameter is not null and throws an exception if it is.
+        /// </summary>
+        protected void CheckSettingsIsNotNull(SvnRemoteSettings settings)
+        {
+            if (settings == null)
+            {
+                throw new ArgumentNullException(nameof(settings));
+            }
+
```

Comment:
nit: remove empty row

Reply:
That was easy. I've just pushed an update to fix it.